examples/rules_ios/MODULE.bazel

```diff
@@ -20,7 +20,7 @@ bazel_dep(
 )
 bazel_dep(
     name = "rules_ios",
-    version = "3.2.1",
```

Comment:
Annoying nit: can you bump this in its own PR?

Reply:
Sure ill get 4.0.0 updated once thats released